src/components/textareaInput/TextareaInput.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from "./textareaInput.module.css"
+
+export const TextAreaInput = ({
```

Comment:
maybe a little nitpicky 😂 component name is not consistent with file naming, could we rename file to match the component?